### ISSUE TITLE
The n_batch parameter doesn't seem to work anymore.

### ISSUE THREAD

**Issue by BadisG** [oobabooga/text-generation-webui#5133]
### Describe the bug

Hello,

When playing with Mixtral I noticed that the model got suddently dumber in the last few days, for example during the chat the bot doesn't understand me anymore and starts to get into infinite loop (repeating its last sentence over and over).

I think I found the issue of it, we know that CuBlas is non deterministic when we change the batch size https://github.com/oobabooga/text-generation-webui/issues/4478#issue-1977489970, hence why I usually go for n_batch = 2048 to get a smarter model.

But now, changing the n_batch doesn't change the logits at all, it's as if it's fixed to a single value (maybe 512) and refuses to go further 
![C3dQ8QH](https://github.com/oobabooga/text-generation-webui/assets/110173477/32d36525-3f9a-47aa-bc02-8a2e8a634718)

Maybe some previous commit changed something Idk, but this need to be fixed, I'm a bit sad to see my Mixtral struggle like that ;_;

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Change the n_batch and see the logits (should be different but it's not anymore)

### Logs

```shell
No logs.
```

### System Info

```shell
Windows 10
Model: Mixtral-instruct
Offloading: 8 layers in the GPU
```
